instaseis/scripts/repack_db.py

```diff
@@ -490,7 +490,11 @@ def repack_database(input_folder, output_folder, contiguous,
 
     assert found_filenames, "No files named `ordered_output.nc4` found."
 
-    os.makedirs(output_folder)
+    if input_folder == output_folder:
```

Comment:
the variables input_folder and output_folder are used afterwards, is it save to overwrite them here? I guess so...